src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -944,7 +944,8 @@ public override Stream OpenRead(string resolvedPath)
             catch (Exception ex)
             {
                 // line information is only available when PDBs have been emitted
-                var stackTrace = new StackTrace(ex, needFileInfo: true);
+                var needFileInfo = true;
```

Comment:
The parameter name to `StackTrace` is different between different TF: needFileInfo vs. fneedFileInfo. Hence can't pass by name here and need to use a local to create the same effect.